docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc

```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
+. Click image:6176.png[] next to the *(options)* label to open the *Add an option to the rule* window.
+. Select *ruleflow-group* from the *Attribute* drop-down menu and enter *validation* in the *ruleflow-group* field.
+. Click *Save*, then click *Save* to confirm your changes.
 
 == Creating Guided Decision Tables
 
 To create a guided decision table:
 
 . Log into Business Central. Click *Menu* -> *Projects*, then *process-app-start*.
-. Click *Create new asset* -> *Guided Decision Table*, then enter the following:
+. Click *Create New Asset* -> *Guided Decision Table*, then enter:
 +
 * *Name*: `Mortgage Decision Table`
 * *Package*: `org.jboss.example`
 * Check *Use Wizard*
 +
-Click *Ok*. The *Guided Decision Table Wizard* dialogue opens. 
+Click *+Ok*. The *Guided Decision Table Wizard* window opens.
 
-. Click *Next*, then *Next*. All the objects from the same package are imported by default.
-. In the *Define Facts\Patterns on which constraints can be defined* part, place *Applicant* and *Property* facts to the *Chosen patterns* window. Then, click *Next*.
-. In the *Define constraints on the Facts\Patterns fields* part:
-.. From *Available patterns*, select *Applicant*. From *Available fields*, select *annualincome*, and place it into *Conditions*. 
-.. From *Conditions*, select *annualincome*, then enter the following:
+. Select *Add Fact Patterns*.
+. Move the *Applicant* and *Property* patterns to the *Chosen patterns* section.
+. Click *Add Constraints*, select *Applicant > annualincome:Whole number (interger)*, and move *annualincome..* to the *Conditions* section.
+. From *Conditions*, select *annualincome*, then enter:
 +
 * *Column header (description)*: `Annual Income (Greater Than)`
 * *Operator*: `greater than`
-.. From *Available fields*, select *annualincome* and, place it into *Conditions*.
-.. From *Conditions*, select *annualincome*, then enter the following:
 +
-* *Column header (description)*: `Annual Income (Less Than Or Equal To)`
+. Move *annualincome..* to the *Conditions* section again.
+. From *Conditions*, select *annualincome*, then enter:
++
+* *Column header (description)*: `Annual Income (Less Than or Equal To)`
 * *Operator*: `less than or equal to`
-.. From *Available patterns*, select *Property*. From *Available fields*, select *saleprice*, and place it into *Conditions*.
-.. From *Conditions*, select *saleprice*, then enter the following: 
++
+. From *Available patterns*, select *Property*,  then from *Available fields*, select *saleprice:Whole number (interger)* and then move it to the *Conditions* section.
+. From *Conditions*, select *saleprice*, then enter:
 +
 * *Column header (description)*: `Sale Price (Less Than)`
 * *Operator*: `less than`
-.. From *Available fields*, select *age*, and place it into *Conditions*. 
-.. From *Conditions*, select *age*, then enter the following: 
+. From *Available fields*, select *age:Whole number (interger)*, and move it in to the *Conditions* section.
```

Comment:
Can this just be "move it to the *Conditions* section"? I'm not actually sure if that makes sense without seeing it, ignore me if it doesn't.